charts/dafka-consumer/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.alerting.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  {{- if .Values.alerting.labels }}
+  labels:
+  {{ toYaml .Values.alerting.labels | nindent 4 }}
+  {{- end }}
+  name: {{ .Release.Name }}-{{ .Values.name }}
+spec:
+  groups:
+  - name: ./{{ .Release.Name }}-{{ .Values.name }}
+    rules:
+    - alert: Kafka Consumer Group Missing
+      expr: absent(kafka_consumer_group_offset{name="{{ .Values.groupId | default .Release.Name }}"})
```

Comment:
where does this metric come from?

Reply:
@omerlh is this a specific Aiven metric? or something dafka-consumer itself expose?
If it can be different across providers then it should be configurable

Reply:
Look like those are Aiven metrics but I think it's not coming from them - I think it's exported by Kafka by default

Reply:
I don't think its exported from the kafka library itself, so I think its better to inejct the `rules`